src/main/java/ch/aaap/harvestclient/impl/TeamReportApiImpl.java

```diff
@@ -0,0 +1,36 @@
+package ch.aaap.harvestclient.impl;
+
+import ch.aaap.harvestclient.api.TeamReportsApi;
+import ch.aaap.harvestclient.api.filter.TeamReportFilter;
+import ch.aaap.harvestclient.domain.TeamReport;
+import ch.aaap.harvestclient.domain.pagination.PaginatedList;
+import ch.aaap.harvestclient.domain.pagination.Pagination;
+import ch.aaap.harvestclient.service.TeamReportService;
+import java.util.List;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import retrofit2.Call;
+
+public class TeamReportApiImpl implements TeamReportsApi {
+
+  private final TeamReportService teamReportService;
+  private static final Logger log = LoggerFactory.getLogger(RolesApiImpl.class);
+
+  public TeamReportApiImpl(TeamReportService teamReportService) {
+    this.teamReportService = teamReportService;
+  }
+
+  @Override
+  public List<TeamReport> list(TeamReportFilter filter) {
+    log.debug("Getting team report for period {} - {}", filter.getFrom(), filter.getTo());
```

Comment:
API docs mention that from - to needs to be at most 1 year, what happens when it's not? Do we react nicely?
Otherwise we could enforce that here

Reply:
For now, I predict just throwing bad request exception. :)